### PR TITLE
Loading gif commented.

### PR DESCRIPTION
**Pull request by AndreeaEne**

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -17,7 +17,8 @@
       left: calc(50% - 150px);
       top: calc(50% - 40px);
     ">
-      <img src="loading.gif" style="width: 300px;"/><br>
+      <!--Loading gif when the application starts-->
+      <!--<img src="loading.gif" style="width: 300px;"/><br>-->
       <center>Loading...</center>
```
Comment (captainsafia) [inline static/index.html]: Since the code is version-controlled, we can actually just go ahead and delete lines 20 through 22.

**Review by captainsafia — CHANGES_REQUESTED** [review]

**Comment by captainsafia** [comment]
Thanks for the pull request, @AndreeaEne, and welcome to the nteract community!

I posted a quick comment for you to update the code.

File: static/index.html (hunk 1 above)
Comment (AndreeaEne) [reply static/index.html]: 22nd included?

Comment (captainsafia) [reply static/index.html]: Yes! I actually just realized you can delete the entire div. So you can remove lines 13 to 23.

**Comment by captainsafia** [comment]
Also, we follow the [conventional-changelog-standard](https://github.com/bcoe/conventional-changelog-standard/blob/master/convention.md) for writing our commit messages.

Do you have experience editing existing commit messages to comply to the standard we use or would you like me to edit the commit messages once you've made your updates?

**Comment by AndreeaEne** [comment]
@captainsafia, can you please edit the commit message, I'd rather see the way you're writing for the future commit changes. Thanks :)

**Review by captainsafia — APPROVED** [review]

**Comment by captainsafia** [comment]
> @captainsafia, can you please edit the commit message, I'd rather see the way you're writing for the future commit changes. Thanks :)

No problem I can do that! If you'd like to see how we format our commit messages, you can take a look at [our commit log](https://github.com/nteract/nteract/commits/master).

**Comment by rgbkrk** [comment]
Thanks for removing this leftover!

**Pull request merged**